build/build.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
+VisualStudioVersion = 15.0.26206.0
```

Comment:
it is. This doesn't actually matter. I built in 2017 for funsies.

---
In reply to: [102594106](https://github.com/Microsoft/WinObjC/pull/2034#discussion_r102594106) [](ancestors = 102594106)